toggl_hammer/toggl_hammer.py

```diff
@@ -91,21 +91,29 @@ def entry_prompt(self, date_index):
             print("[{0}] {1}".format(i, project[0]))
         proj_index = input("choose project: ")
         proj_index = int(proj_index)
+        proj_id = self.projects[proj_index][1]
+        task_list = requests.get('https://www.toggl.com/api/v8/projects/' + str(proj_id) + '/tasks', auth=(API_KEY, 'api_token')).json()
+        for i, task in enumerate(task_list):
```

Comment:
Does not account for `task_list == None`, which is what you get if there are no tasks.  Maybe only if your account doesn't support them?  Not sure.